src/include/optimizer/statistics/stats_util.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include <limits>
+
+#include "loggers/optimizer_logger.h"
+#include "type/transient_value.h"
+#include "type/transient_value_peeker.h"
+
+namespace terrier::optimizer {
+
+/**
+ * Statistics utilities for Selectivity
+ */
+class StatsUtil {
+ public:
+  /**
+   * Converts numeric TransientValue type to primitive value
+   * Return NaN if value is not numeric
+   * @param value TransientValue to convert
+   * @returns primitive value or NaN
+   */
+  static double TransientValueToNumericValue(const type::TransientValue &value) {
```

Comment:
If this is the only method in this class, then we should probably just move it to `src/include/type/transient_value_util.h`